src/Compilers/Core/Portable/Generated/Operations.xml.Generated.cs

```diff
@@ -3872,7 +3872,7 @@ internal sealed partial class LazyRangeCaseClause : BaseRangeCaseClause, IRangeC
     internal abstract partial class BaseRelationalCaseClause : CaseClause, IRelationalCaseClause
     {
         public BaseRelationalCaseClause(BinaryOperatorKind relation, CaseKind caseKind, SemanticModel semanticModel, SyntaxNode syntax, ITypeSymbol type, Optional<object> constantValue, bool isImplicit) :
-            base(caseKind, OperationKind.RelationalCaseClause, semanticModel, syntax, type, constantValue, isImplicit)
+            base(caseKind, semanticModel, syntax, type, constantValue, isImplicit)
```

Comment:
>caseKind [](start = 17, length = 8)

What different case kinds can be used with this node?

Reply:
It looks like all CaseKinds can be hardcoded.

---
In reply to: [139253449](https://github.com/dotnet/roslyn/pull/22133#discussion_r139253449) [](ancestors = 139253449)